.eslintrc.js

```diff
@@ -1,275 +1,29 @@
-    "no-plusplus": "off",
-    "no-process-env": "off",
-    "no-process-exit": "error",
-    "no-proto": "error",
-    "no-prototype-builtins": "error",
-    "no-restricted-globals": "error",
-    "no-restricted-imports": "error",
-    "no-restricted-modules": "error",
-    "no-restricted-syntax": "error",
-    "no-return-assign": "error",
-    "no-script-url": "error",
-    "no-self-compare": "error",
-    "no-sequences": "error",
-    "no-shadow": "off",
-    "no-shadow-restricted-names": "error",
-    "no-spaced-func": "error",
-    "no-sync": "error",
-    "no-ternary": "off",
-    "no-throw-literal": "error",
-    "no-undef-init": "error",
-    "no-undefined": "off",
-    "no-underscore-dangle": "off",
-    "no-unmodified-loop-condition": "error",
-    "no-unneeded-ternary": "error",
-    "no-unused-expressions": "off",
-    "no-unused-vars": "warn",
-    "no-use-before-define": "off",
-    "no-useless-call": "error",
-    "no-useless-computed-key": "error",
-    "no-useless-concat": "error",
-    "no-useless-constructor": "error",
-    "no-useless-escape": "off",
-    "no-useless-rename": "error",
-    "no-var": "error",
-    "no-void": "error",
-    "no-warning-comments": "off",
-    "no-whitespace-before-property": "error",
-    "no-with": "error",
-    "object-curly-spacing": [
-      "warn",
-      "always"
-    ],
-    "object-property-newline": [
-      "error",
-      {
-        "allowMultiplePropertiesPerLine": true
-      }
-    ],
-    "one-var": "off",
-    "one-var-declaration-per-line": "error",
-    "operator-assignment": "error",
-    "operator-linebreak": "error",
-    "padded-blocks": "off",
-    "prefer-arrow-callback": "off",
-    "prefer-const": "off",
-    "prefer-reflect": "off",
-    "prefer-rest-params": "error",
-    "prefer-spread": "error",
-    "prefer-template": "off",
-    "quote-props": "off",
-    "quotes": [
-      "error",
-      "single"
-    ],
-    "no-this-before-super": "off",
-    "radix": "error",
-    "rest-spread-spacing": "error",
-    "semi": ["warn", "always"],
-    "semi-spacing": "error",
-    "sort-imports": "off",
-    "sort-vars": "error",
-    "space-before-blocks": "error",
-    "space-before-function-paren": "off",
-    "space-in-parens": [
-      "error",
-      "never"
-    ],
-    "space-infix-ops": "error",
-    "space-unary-ops": "error",
-    "spaced-comment": [
-      "error",
-      "always"
-    ],
-    "strict": "error",
-    "template-curly-spacing": [
-      "error",
-      "never"
-    ],
-    "unicode-bom": [
-      "error",
-      "never"
-    ],
-    "valid-jsdoc": "off",
-    "vars-on-top": "error",
-    "wrap-iife": "error",
-    "wrap-regex": "off",
-    "yield-star-spacing": "error",
-    "yoda": [
-      "error",
-      "never"
-    ]
+  rules: {
+    indent: ['error', 2, { SwitchCase: 1 }],
+    'linebreak-style': ['error', 'unix'],
+    quotes: ['error', 'single'],
+    semi: ['warn', 'always'],
+    'no-console': 'off',
+    'no-unused-vars': 'warn',
```

Comment:
Do not you want to error on unused vars?

Reply:
same ⬆️

Reply:
Same answer for the next comment, different scope, if you agree please create a task if not then lets discuss it later.